.travis.yml

```diff
@@ -32,3 +32,11 @@ script:
   - docker-compose logs
   - curl --retry 10 --retry-delay 5 -D- http://localhost:9200/
   - curl --retry 10 --retry-delay 5 -D- http://localhost:5601/api/status
+  - docker-compose down && docker-compose rm
+
+  # Test docker-swarm.yml
+  - docker swarm init
```

Comment:
Cannot confirm that this will work, @antoineco thoughts?

Reply:
It should, at least the syntax is correct. I know nothing about docker stacks though. Isn't the file format supposed to be different from Compose?

Reply:
All good then.

Actually it's the same file format but not the same version (`docker-compose` uses `2.x` and `docker stack` uses `3.x`).
See the documentation here: https://docs.docker.com/compose/compose-file/

Reply:
I got mixed up with Cloud stacks then
https://docs.docker.com/docker-cloud/apps/stack-yaml-reference/